### PR TITLE
batch fix fixable flaws in web/api/

### PR DESCRIPTION
**Pull request by peterbe**
Trying to get rid of all the `/@api/deki` images. But limiting it to as few pages per batch as possible.

### CODE REVIEW

**Comment by peterbe** [comment]
@nschonni As a sanity check, I made this here PR by using the code from https://github.com/mdn/yari/pull/2660

File: files/en-us/web/api/window/innerheight/index.html [hunk 1]
```diff
@@ -77,7 +77,7 @@ <h3 id="Graphical_example">Graphical example</h3>
   <code>innerHeight</code>.</p>
 
 <p><img alt="innerHeight vs outerHeight illustration"
-    src="/@api/deki/files/213/=FirefoxInnerVsOuterHeight2.png"></p>
+    src="firefoxinnervsouterheight2.png"></p>
```
Comment (Rumyra) [inline files/en-us/web/api/window/innerheight/index.html:80]: May this image forever stay

**Review by Rumyra — APPROVED** [review]
I've checked out a few pages and the images look fine & well 👍

I'm hoping someone with more authority will press the big green button to merge however 🟩 - or at least sanity check my approval

**Review by chrisdavidmills — APPROVED** [review]
This is fine; it's automated fixable flaw stuff.

**Pull request merged**